core/src/apps/common/signverify.py

```diff
@@ -4,9 +4,9 @@
 from trezor.crypto.hashlib import blake256, sha256
 from trezor.ui.text import Text
 
+from apps.bitcoin.sign_tx.writers import write_varint
```

Comment:
maybe we should move `write_varint` to `apps.common.writers`?

maybe rename it to `write_bitcoin_varint` to differentiate from protobuf varint?

Reply:
Done in e5856b3e6ed7ee092a6dc3b280ddf35d9ba605c4.